### PR TITLE
Add `set -o pipefail` to CI script

### PR DESCRIPTION
**Pull request by lukasz-szyszkowski**

### CODE REVIEW

**Review by ben-xD — APPROVED** [review]

**Review by maratal — APPROVED** [review]

File: Scripts/assemble.sh [hunk 1]
```diff
@@ -10,39 +10,39 @@ echo
 echo '\033[1mBuild: AblyAssetTrackingSubscriber Library\033[0m'
 echo
 
-xcodebuild -scheme "AblyAssetTrackingPublisher" -destination "generic/platform=iOS" SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES | xcpretty
+set -o pipefail & xcodebuild -scheme "AblyAssetTrackingPublisher" -destination "generic/platform=iOS" SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES | xcpretty
```
Comment (lawrence-forooghian) [inline Scripts/assemble.sh:13]: Would it not be sufficient to put `set -o pipefail` once at the top of the file, below `set -e`?

Comment (lukasz-szyszkowski) [reply Scripts/assemble.sh:13]: yes, once should be ok, I'll change it

**Review by lawrence-forooghian — APPROVED** [review]

File: Scripts/assemble.sh (hunk 1 above)
Comment (lukasz-szyszkowski) [reply Scripts/assemble.sh]: @lawrence-forooghian if t's ok, please resolve this conversation. Thanks

Comment (lawrence-forooghian) [reply Scripts/assemble.sh]: 👍

**Pull request merged**